install.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/sh -e
 
+SUDO=`which sudo`
+
+if [ "`find "$SUDO" -perm -4000`" = "$SUDO" ]; then
```

Comment:
The binary `sudo` is able to be execute in both types of images, iirc. But only one has the setuid bit set.